src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingService.kt

```diff
@@ -75,7 +76,8 @@ class EksternBehandlingService(
         val revurdering = RevurderingDto(
             fagsakId = fagsak.id,
             behandlingsårsak = BehandlingÅrsak.KLAGE,
-            kravMottatt = LocalDate.now()
+            kravMottatt = LocalDate.now(),
+            vilkårsbehandleNyeBarn = VilkårsbehandleNyeBarn.VILKÅRSBEHANDLE
```

Comment:
Hvis vi kun ønsker å løse det med at barn forsvinner i revurdering fra klage, så går det å kun sende med nye barn fra denne. Dvs hvis vi kun ønsker å løse https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=NAV-11926 og ikke forholde seg til disse enums

Reply:
Fint sånn som det er gjort 👍

Reply:
Den som heter `automatiskJournalfør` i `JournalføringService` bør kanskje også sende med "VILKÅRsBEHANDLE" ?

Reply:
Hmm ja, samtidig blir den egentlige ikke brukt i den løypa. Lurer egentlige på om man skulle bytte ut litt av ette med sealed classes for å få en mer typsikkret flyt 👀